src/library/dao/trackdao.cpp

```diff
@@ -277,8 +277,8 @@ void TrackDAO::saveTrack(Track* pTrack) const {
         return;
     }
     qDebug() << "TrackDAO: Saving track"
-            << trackId
-            << pTrack->getFileInfo();
+             << trackId
```

Comment:
This indentation does not conform to clang-format which is using a fixed indentation of 2 * 4 = 8 spaces and not aligning vertically. Please revert all unintended changes.

Reply:
This is what clang-format is doing alone :(

Reply:
When I revert the changes they get reapplied by the pre-commit hook.